### DISCUSSION TITLE
Power limit for 4000 Series

### DISCUSSION THREAD

**Discussion by MckenzieDavis** [seerge/g-helper#649] [Q&A] [answered]
Hi, I have 2020 Zephyrus which is 4800hs, Is the set power limit not available for 2020 Zephyrus?, Cause its possible to set power limit UXTU, I don't use UXTU anymore because it's starting to bloat with a lot of features that I dont use.

![Screenshot 2023-06-20 023737](https://github.com/seerge/g-helper/assets/82803124/28bd992f-8cf2-4228-b78f-0aaad525a431)

**Comment by seerge** [comment, marked as answer]
G-helper uses asus own endpoints to set power limits, if they are not visible - it means that your device don't support them :| (at least in asus way). I may add some alternative ways in future, but i'm also not sure as i don't want to overbloat app with features :)
